package.json

```diff
@@ -40,7 +40,7 @@
     "test:e2e-matter": "node src-script/zap-uitest.js open matter",
     "test:e2e-matter-ci": "node src-script/zap-uitest.js run matter",
     "report": "node src-script/zap-combine-reports.js",
-    "postinstall": "exec electron-builder install-app-deps && husky install && npm rebuild canvas --update-binary && npm run version-stamp",
+    "postinstall": "exec husky install && npm rebuild canvas --update-binary && npm run version-stamp",
```

Comment:
It installs sqlite3 version 5.1.6 which causes an error when creating the Windows binary.

Reply:
Is post install called from somewhere else as well? I am wondering if this was called during `npm install` or `npm ci`? Would the source installation be affected because of this in any way?

Reply:
I don't think so since it passed all the tests

Reply:
> To ensure your native dependencies always matched the electron version, simply add script "postinstall": "electron-builder install-app-deps" to your package.json.

That's the phrase from https://www.electron.build/ 

We should keep this as it's useful for any user "install" when package.json is changed. 
Does this prevent the mac os from creating the Windows binary? 
If that's the case, we should dig in further.

Reply:
@jingteng25742 https://www.electron.build/tutorials/two-package-structure.html since we do not use the two package.json structure whatever is in our package.json will be installed.

Reply:
The problem is that sqlite3 5.1.6 causes electron builder an error when creating the Windows binary from a Mac or from linux. sqlite5.1.4 works. That line "electron-builder install-app-deps" causes sqlite3 5.1.6 to be installed no matter what is defined in package.json. Taking out the line gives us more flexibility.

Reply:
thank you, I will do that

Reply:
@jingteng25742 let me know if you see anything else